tests/internal/e2e/test/specs/vcast_build_env_failure.test.ts

```diff
@@ -0,0 +1,242 @@
+    );
+  });
+
+  it("should activate vcastAdapter", async () => {
+    await updateTestID();
+
+    await browser.keys([Key.Control, Key.Shift, "p"]);
+    // Typing Vector in the quick input box
+    // This brings up VectorCAST Test Explorer: Configure
+    // so just need to hit Enter to activate
+    for (const character of "vector") {
+      await browser.keys(character);
+    }
+
+    await browser.keys(Key.Enter);
+
+    const activityBar = workbench.getActivityBar();
+    const viewControls = await activityBar.getViewControls();
+    for (const viewControl of viewControls) {
+      console.log(await viewControl.getTitle());
+    }
+
+    await bottomBar.toggle(true);
+    const outputView = await bottomBar.openOutputView();
+
+    // Await last expected sentence
+    await browser.waitUntil(
+      async () =>
+        (await outputView.getText())
+          .toString()
+          .includes("Please refer to the installation and configuration instructions for details on resolving these issues"),
+      { timeout: TIMEOUT }
+    );
+  });
+
+  it("should throw an error on invalid PATH env", async () => {
+    await updateTestID();
+    const workbench = await browser.getWorkbench();
+    const activityBar = workbench.getActivityBar();
+    const explorerView = await activityBar.getViewControl("Explorer");
+    await explorerView?.openView();
+
+    // Put in invalid path in settings
+    const settingsEditor = await workbench.openSettings();
+    const unitTestLocationSetting = await settingsEditor.findSetting(
+      "Vectorcast Installation Location",
+      "Vectorcast Test Explorer"
+    );
+    await unitTestLocationSetting.setValue("some/invalid/path");
+
+    await (await workbench.openNotificationsCenter()).clearAllNotifications();
+
+    await bottomBar.toggle(true);
+    const outputView = await bottomBar.openOutputView();
+
+    // Await last expected sentence
+    await browser.waitUntil(
+      async () =>
+        (await outputView.getText())
+          .toString()
+          .includes("Please refer to the installation and configuration instructions for details on resolving these issues"),
+      { timeout: TIMEOUT }
+    );
+  });
+
+
+  it("should create VectorCAST environment and fail", async () => {
+    await updateTestID();
+
+    const workbench = await browser.getWorkbench();
+    const activityBar = workbench.getActivityBar();
+    const explorerView = await activityBar.getViewControl("Explorer");
+    await explorerView?.openView();
+
+    // Expand vcastTutorial and cpp folder
+    const workspaceFolderSection =
+      await expandWorkspaceFolderSectionInExplorer("vcastTutorial");
+    const cppFolder = workspaceFolderSection.findItem("cpp");
+    await (await cppFolder).select();
+
+    // Get file and "Create VectorCAST Environment"
+    const managerCpp = await workspaceFolderSection.findItem("manager.cpp");
+    await managerCpp.select()
+    console.log("Selected File: " + managerCpp)
```

Comment:
`xo`:

- `managerCpp will evaluate to [object Object] when stringified.`
- `Insert ;`